ci/common/common.sh

```diff
@@ -104,28 +119,31 @@ commit_subtree() {(
 
   git add --all .
 
-  [[ ${CI} == true ]] && {
+  if is_ci_build --silent ; then
     # Commit on Travis CI.
-    if [[ -z "${GH_TOKEN}" ]]; then
-      echo "GH_TOKEN not set, not committing."
-      echo "To test pull requests, see instructions in README.md."
-      return 1
-    fi
-
     git config --local user.name ${GIT_NAME}
     git config --local user.email ${GIT_EMAIL}
 
     git commit -m "${CI_TARGET//-/ }: Automatic update." || true
+
+    if test -z "${GH_TOKEN}" ; then
```

Comment:
`-u` is complaining about GH_TOKEN.

    /home/travis/build/neovim/bot-ci/ci/common/common.sh: line 129: GH_TOKEN: unbound variable

Do we care? pvs-report.sh seems to be working otherwise.